tests/templates/kuttl/logging/05-deploy-automatic-log-config-spark-app.yaml.j2

```diff
@@ -5,13 +5,15 @@ metadata:
   name: spark-automatic-log-config
 spec:
   version: "1.0"
-  sparkImage: docker.stackable.tech/stackable/spark-k8s:{{ test_scenario['values']['spark'].split('-stackable')[0] }}-stackable{{ test_scenario['values']['spark'].split('-stackable')[1] }}
-  sparkImagePullPolicy: IfNotPresent
+  sparkImage: docker.stackable.tech/sandbox/sbernauer/spark-k8s:{{ test_scenario['values']['spark'].split('-stackable')[0] }}-stackable{{ test_scenario['values']['spark'].split('-stackable')[1] }}
```

Comment:
Will revert it back to docker.stackable.tech/stackable/spark-k8s once images have been merged